web/client/plugins/Maps.jsx

```diff
@@ -18,10 +19,12 @@ const {mapTypeSelector} = require('../selectors/maptype');
 const {userRoleSelector} = require('../selectors/security');
 const { totalCountSelector } = require('../selectors/maps');
 const { isFeaturedMapsEnabled } = require('../selectors/featuredmaps');
```

Comment:
this selector has no test, can you add it ?

Reply:
@MV88 we have a similar background for the empty dashboard section, so it is respecting the current L&F. We will review both background in separated issue if needed later.

Reply:
I didn't create this selector though.  I was following the rule that requires a developer to focus on the files associated with the issue he/she currently working on it.

Reply:
i know, since you are there you can add a small test for a piece of code you are re-using

Reply:
> @MV88 we have a similar background for the empty dashboard section, so it is respecting the current L&F. We will review both backgrounds in separated issue if needed later.

ok

Reply:
> i know, since you are there you can add a small test for a piece of code you are re-using

Ok...!